lib/combobox.js

```diff
@@ -22,7 +22,6 @@ module.exports = React.createClass({
      * ```
     */
     onInput: React.PropTypes.func,
-
```

Comment:
We should respect the formatting of the original library to prevent future merge conflicts